tests/all_tests.rb

```diff
@@ -1,8 +1,9 @@
-Dir.chdir(File.dirname(__FILE__)) do
-  Dir.glob('**/test_*.rb') do |test_case|
-    require "#{File.expand_path(File.dirname(__FILE__))}/#{test_case}"
-  end
+Dir.glob('tests/**/test_*.rb') do |test_case|
+  require File.expand_path(test_case)
 end
 
-# To run a single test:
-# require_relative 'units/test_lib_meets_required_version'
+# To run a single test, comment the above and uncomment:
+# require 'tests/units/test_lib_meets_required_version'
```

Comment:
I think we should REMOVE the recommendation in the comment above to edit this file to run a single test and just leave the comment below about running a single test from the command line.